web_monitoring/differs.py

```diff
@@ -101,9 +102,15 @@ def html_source_diff(a_text, b_text):
     TIMELIMIT = 2 #seconds
     return compute_dmp_diff(a_text, b_text, timelimit=TIMELIMIT)
 
+
 def html_diff_render(a_text, b_text):
     """
-    HTML Diff for rendering
+    HTML Diff for rendering.
+
+    Please note that the result of this should not be displayed as-is in a
+    browser -- because this contains added and removed sections of the
+    document’s <head>, it may cause a browser to load two different CSS or JS
+    files that are in conflict with each other.
```

Comment:
Just to be clear, this is not a change in behavior—this was already the case. I wanted to make sure it was extra clear, though.

Reply:
I wanted to talk to you about this, especially after I saw the result of the diff between the examples mentioned in issue #91.
Important to have this in the docs.